lib/fastlane/plugin/wpmreleasetoolkit/actions/common/buildkite_trigger_build_action.rb

```diff
@@ -0,0 +1,90 @@
+module Fastlane
+  module Actions
+    class BuildkiteTriggerBuildAction < Action
+      def self.run(params)
+        require 'buildkit'
+
+        UI.message "Triggering build on branch #{params[:branch]} at #{params[:commit]}"
+
+        pipeline_name = {
+          PIPELINE: params[:pipeline_file]
+        }
+
+        client = Buildkit.new(token: params[:buildkite_token])
+        response = client.create_build(
+          params[:buildkite_organization],
+          params[:buildkite_pipeline],
+          {
+            branch: params[:branch],
+            commit: params[:commit],
+            env: params[:environment].merge(pipeline_name)
+          }
+        )
+
+        response.state == 'scheduled' ? UI.message('Done!') : UI.user_error!("Failed to start job\nError: [#{response}]")
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        'Triggers a job on Buildkite'
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(
+            key: :buildkite_token,
+            env_name: 'BUILDKITE_ACCESS_TOKEN',
+            description: 'Buildkite Personal Access Token',
+            type: String
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :buildkite_organization,
+            env_name: 'BUILDKITE_ORGANIZTION',
+            description: 'The Buildkite organization that contains your pipeline',
+            type: String,
+            default_value: 'wordpress-mobile'
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :buildkite_pipeline,
+            env_name: 'BUILDKITE_PIPELINE',
+            description: 'The Buildkite pipeline you\'d like to build',
```

Comment:
I wasn't expecting that escape to work, but it does 😄 

```
➜ irb
irb(main):001:0> s = 'The Buildkite pipeline you\'d like to build'
=> "The Buildkite pipeline you'd like to build"
```

Reply:
Alternatively, you could use:
```suggestion
            description: %(The Buildkite pipeline you'd like to build),
```
(but it's just a matter of preference, really, so 🤷 )

Reply:
Ah this is better